cql3/restrictions/single_column_primary_key_restrictions.hh

```diff
@@ -46,6 +46,8 @@
 #include "cartesian_product.hh"
 #include "cql3/restrictions/primary_key_restrictions.hh"
 #include "cql3/restrictions/single_column_restrictions.hh"
+#include "restrictions_config.hh"
```

Comment:
`cql3/cql_config.hh` already includes this, why does it need to be included again here?

Reply:
I can remove it.